core/src/Platforms/iOS/TokenCacheAccessor.cs

```diff
@@ -32,90 +32,248 @@
 using Microsoft.Identity.Core;
 using Microsoft.Identity.Core.Cache;
 using System.Collections.ObjectModel;
+using Microsoft.Identity.Core.Helpers;
 
 namespace Microsoft.Identity.Core
 {
     internal class TokenCacheAccessor : ITokenCacheAccessor
     {
-        private const string AccessTokenServiceId = "com.microsoft.identity.client.accessToken";
-        private const string RefreshTokenServiceId = "com.microsoft.identity.client.refreshToken";
-        private const string IdTokenServiceId = "com.microsoft.identity.client.idToken";
-        private const string AccountServiceId = "com.microsoft.identity.client.Account";
+        public const string CacheKeyDelimiter = "-";
+
+        static Dictionary<string, int> AuthorityTypeToAttrType = new Dictionary<string, int>()
```

Comment:
Please specify culture (invariant, case insenstive)

Reply:
What are these magic numbers?

Reply:
is this needed here?

---
In reply to: [199550365](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1108#discussion_r199550365) [](ancestors = 199550365)